docs/bibliography.bib

```diff
@@ -315,3 +315,12 @@ @article{wilson:2017
    pages = {e1005510},
    doi = {10.1371/journal.pcbi.1005510}
 }
+@article{verscharen:2019,
+   title = {The multi-scale nature of the solar wind},
+   volume = {16},
+   doi = {10.1007/s41116-019-0021-0},
+   number = {1},
+   journal = {Living Reviews in Solar Physics},
+   author={Verscharen, Daniel and Klein, Kristopher G. and Maruca, Bennett A.},
+   year={2019}
+}
```

Comment:
```suggestion
```

Also remove this since it's associated with the new functionality you will be adding in the followup PR.